app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -623,6 +623,26 @@ $after-button-padding-left: govuk-spacing(4);
 
   &.gem-c-layout-super-navigation-header__search-toggle-button--blue-background {
     background: $govuk-brand-colour;
+
+    &:hover {
+      color: govuk-colour("white");
+
+      &::after {
+        background: govuk-colour("white");
+      }
+    }
+
+    @include focus-not-focus-visible {
+      &::after {
+        background: none;
+      }
+
+      &:hover {
+        &::after {
+          background: govuk-colour("white");
+        }
+      }
+    }
```

Comment:
I think this mixin might be generating some duplicate styles:
```
.gem-c-layout-super-navigation-header__search-toggle-button
.gem-c-layout-super-navigation-header__search-toggle-button--blue-background:hover::after {
  background: #fff;
}

.gem-c-layout-super-navigation-header__search-toggle-button
.gem-c-layout-super-navigation-header__search-toggle-button--blue-background:hover::after {
  background: #fff;
}
```

Reply:
I agree - the duplicate styles aren’t causing any issues and have a negligible impact on file size, so it makes sense to add this to our backlog and address it at a later stage